src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -497,6 +498,16 @@ class BugsnagPlugin : Plugin<Project> {
         }
     }
 
+    internal fun getSourcemapUploadEndpoint(bugsnag: BugsnagPluginExtension): String {
+        return when (val endpoint = bugsnag.endpoint.get()) {
+            UPLOAD_ENDPOINT_DEFAULT -> endpoint
+            else -> {
+                val host = URL(endpoint)
+                return URL(host, "sourcemap").toString()
```

Comment:
My bad, I told you it was this. It should actually be the RN path:

```suggestion
                return URL(host, "react-native-source-map").toString()
```